### PR TITLE
Make rebind of CommandPalette optional

### PR DESCRIPTION
**Pull request by planger**
Fixes https://github.com/eclipse-glsp/glsp/issues/188

### CODE REVIEW

File: packages/theia-integration/src/browser/diagram/glsp-theia-diagram-configuration.ts [hunk 1]
```diff
@@ -61,7 +61,9 @@ export abstract class GLSPTheiaDiagramConfiguration implements DiagramConfigurat
         container.bind(OpenerService).toConstantValue(this.openerService);
         container.bind(CommandService).toConstantValue(this.commandService);
         container.bind(ExternalModelSourceChangedHandler).toConstantValue(this.modelSourceChangedHandler);
-        container.rebind(CommandPalette).to(TheiaCommandPalette);
+        if (container.isBound(CommandPalette)) {
+            container.rebind(CommandPalette).to(TheiaCommandPalette);
+        }
```
Comment (martin-fleck-at) [inline packages/theia-integration/src/browser/diagram/glsp-theia-diagram-configuration.ts:66]: Looks good to me but shouldn't we do a "normal" bind if it is not already bound?

Comment (tortmayr) [reply packages/theia-integration/src/browser/diagram/glsp-theia-diagram-configuration.ts:66]: Agreed

Comment (planger) [reply packages/theia-integration/src/browser/diagram/glsp-theia-diagram-configuration.ts:66]: No, I think it doesn't really make sense to bind it if the `CommandPalette` isn't bound. If it isn't bound, the diagram editor doesn't want to have one, so we shouldn't then bind the `TheiaCommandPalette` either. Wdyt?

Comment (martin-fleck-at) [reply packages/theia-integration/src/browser/diagram/glsp-theia-diagram-configuration.ts:66]: Ah right, sorry, I somehow was not thinking about that! You are absolutely right!

**Review by martin-fleck-at — CHANGES_REQUESTED** [review]
I retested this change with the current master and everything still works very well. It seems that the popup now also vanishes already when opening the context menu but this may also be due to  the Theia update. Nevertheless, everything still works fine, please just remove the last commit as the yarn.lock on master is now the correct one.

**Comment by planger** [comment]
Thanks @martin-fleck-at! I force-pushed the single-commit without the yarn.lock change.

**Review by martin-fleck-at — APPROVED** [review]
Great, thank you very much Philip!

**Pull request merged**